doc/user_guide/user-guide.md

```diff
@@ -0,0 +1,56 @@
+# Exasol REST API
+
+## Installation
+
+### Database Side Preparation
+
+* Create a user in the Exasol database for the api service. This user will be used for all connections. Grant **
+  minimum** privileges to the user. We strongly recommend to only grant SELECT on required schemas/tables. Please keep
+  in mind that all API users uses the same service account.
+
+```sql
+CREATE USER api_service_account IDENTIFIED BY secret_password;
+GRANT CREATE SESSION TO api_service_account;
+GRANT SELECT ON SCHEMA my_schema TO api_service_account;
+```
+
+### Configuration File
+
+The application needs a configuration file in yml format. The minimum required parameters are the credentials of the
+Exasol service account described above:
+
+```yaml
+ExasolUser: "someuser"
+ExasolPassword: "somepass"
+```
+
+You can also provide additional configurations:
+
+| Property                  |  Default          | Description                                     |
+| :-----------------------: | :---------------: | :---------------------------------------------- |
+| ApplicationServer         |  "localhost:8080" | Server to start the API application on.         |
```

Comment:
Maybe "ServerAddress" as property name. The description "Address for the server to listen for new connection." looks good (check type errror)

Reply:
The Gin API / `http.ListenAndServe()` call this `addr` ("... listens on the TCP network address addr ..."), so I agree that 'server address' sounds better.

Reply:
About the camel case in yml: it was for correct mapping to the structs because of some trick I had in the code, but now I can probably change it because I changed the implementation. I'm going to try it

Reply:
Changed the documentation description + switched to dases-case in yaml